### ISSUE TITLE
[Suggestion] New shared folders should not be synced automatically

### ISSUE THREAD

**Issue by misida** [owncloud/client#4446]
According to experience most users choose the default "Sync everything from server" option in the sync folder selection dialog after owncloud client installation. It means if someone shares a folder with another user or with a (even a large) group of people the client accepts the new folder and start syncing automatically without any human intervention.
In a large environment ( with thousands of users) where anybody can share folders with anyone else can cause high load on the server side but more importantly large amount of unwanted data downloaded by the clients. As soon as the share is removed from the folder the content will be deleted from the clients computers and if it is shared again sync (download) starts again.

The only solution offered is the sync threshold size. However if the folder is empty at the time of sharing (or smaller than the threshold size if it is set at all) the client accepts the new folder automatically. After that if the folder owner accidentally or purposely uploads huge amount of contents everybody's client who was shared with start syncing 
### Suggested behaviour

When a new shared folder appears in the "root" of the sync folder the client should not "accept" and should not automatically sync independently of the folder's size. Instead the new folder should appear unchecked in the client's folder list and display a warning message similar to the one where the folder size is too big. 
### Actual behaviour

The client "accepts" any new folder if it is empty or its size is smaller than the limit if it is set at all.

ownCloud version:
8.2.2 (stable)

Client version:
2.1.0 stable - 2.2.0git

Thank you in advance

**Comment by AnrDaemon** [comment]
Explicit acceptance of shared object could be a useful uption at times.

**Comment by pmaier1** [comment]
Yes, all incoming shares shall become accept-/rejectable!

**Comment by phil-davis** [comment]
Isn't the suggested behavior here already possible?
![askconfirmation0size](https://cloud.githubusercontent.com/assets/1535615/24649078/936c0aea-1945-11e7-9ef4-fd53d19ca25d.png)
If you set the "folders larger than" limit to 0MB then you get prompted to confirm every new folder.

**Comment by pmaier1** [comment]
> If you set the "folders larger than" limit to 0MB then you get prompted to confirm every new folder.

Good point but currently it does only apply to folders. We want to have accept-/rejectability for all incoming shares (files/folders) to be the (configurable) default on all platforms and in the webUI.
